scripts/xe-reset-networking

```diff
@@ -52,7 +52,7 @@ def write_inventory(inventory):
 def valid_vlan(vlan):
 	if not re.match('^\d+$', vlan):
 		return False
-	if int(vlan)<1 or int(vlan)>=4095:
+	if int(vlan)<0 or int(vlan)>=4095:
```

Comment:
I understand you changed the other condition but are you sure that `>=4095` is correct? I would have expected `>=4096` or `>4095` because 2^12=4096. If it is correct, a comment would be helpful.